README.md

```diff
@@ -86,6 +86,48 @@ resource "aws_instance" "foo" {
 
 **[More examples](./examples)**
 
+## Additional Usage
+
+To handle the occurrence of unique numbers in keys after parsing, use the assertion helper method ```finder```.
+
+tests/tf_assertion_helper_test.py
+```hcl-terraform
+import unittest
+from tf_assertion_helper import finder
```

Comment:
thats not the usage for other people

Reply:
they get to it from `Runner.finder` iirc
so they'd just import runner, then runner.finder....

Reply:
it can be used in this case too I think? Swapping back to runner.finder now